docs/how-to/contribute-docs.rst

```diff
@@ -123,6 +130,37 @@ If a valid spelling of a word is identified as misspelled, then add the word to
 the list in ``docs/spelling_wordlist``. This will add the word to the
 spellchecker's dictionary.
 
+If you get an error related to SSL certificate verification::
+
+    Exception occurred:
+      File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/ssl.py", line 1342, in do_handshake
+        self._sslobj.do_handshake()
+    ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1007)
+
+The root certificate on your machine is out of date. You can correct this by
+installing the Python package `certifi`, and using that package to provide your
+SSL root certificate:
+
+.. tabs::
+
+  .. group-tab:: macOS
+
+    .. code-block:: bash
+
+      (venv) $ pip install certifi
+      (venv) $ export SSL_CERT_FILE=$(python -m certifi)
+
+  .. group-tab:: Linux
+
+    .. code-block:: bash
+
+      (venv) $ pip install certifi
+      (venv) $ export SSL_CERT_FILE=$(python -m certifi)
+
+  .. group-tab:: Windows
+
+    ???
```

Comment:
@rmartin16 Don't suppose you can fill in the gaps here? What's the Windows equivalent of `$()`?

Reply:
Hello
The Windows equivalent is `%()`. For example, if you wanted to set a variable equal to the output of a command in Windows, you could do: `set var=%(command%)`

But I think what you are after is:
``for /f "usebackq delims=" %i in (`python -m certifi`) do set SSL_CERT_FILE=%i``

Reply:
Looks like there's a bunch of installation options for penchant... I think I'll just defer to the official docs.

But: Is pyenchant our only option? AFAICT, the only platform where it seems to work well is Linux...

Reply:
and just so its clear....that `FOR` statement won't run in powershell

Reply:
Understood. I guess I don't have a good feel for what Windows users consider the "default" shell - or how to signal to Windows users (who are, historically, our users with the *least* exposure to the command line) that the command is a Powershell command, not a cmd.exe command.

Reply:
I'm not advocating for PowerShell per se; it introduces its own difficulties. It seems that msft is transitioning to a default experience not based on `cmd.exe`; for instance, they removed it from the Start Menu a while ago. But their approach doesn't seem that heavy-handed, though ( especially when you compare it to how hard they push Edge); so, Windows won't ask me "are you sure you don't want PowerShell even though you typed `cmd` in the search".

Mostly, I'm just interested in making sure we can provide some level of consistency for documenting commands for Windows. For instance, the Android code-signing page contains commands for both cmd and PowerShell.

Reply:
I know this isn't your point but for completeness...

Open PowerShell with Administrative privileges and run:
```powershell
Set-ExecutionPolicy -ExecutionPolicy RemoteSigned
```
and restart your terminal sessions.